tst/EnergyPlus/unit/SurfaceGeometry.unit.cc

```diff
@@ -166,3 +166,67 @@ TEST_F( EnergyPlusFixture, BaseSurfaceRectangularTest )
 	EXPECT_FALSE( ErrorsFound );
 	EXPECT_EQ( Polygonal, Surface( ThisSurf ).Shape );
 }
+
+TEST_F( EnergyPlusFixture, ConfirmCheckSubSurfAzTiltNorm )
+{
+	SurfaceData BaseSurface;
+	SurfaceData SubSurface;
+	bool surfaceError;
+
+	//Case 1 - Base surface and subsurface face the same way - should be no error message and no surfaceError
+	surfaceError = false;
+	BaseSurface.Azimuth = 0.;
+	BaseSurface.Tilt = 0.;
+	BaseSurface.NewellSurfaceNormalVector.x = 0.;
+	BaseSurface.NewellSurfaceNormalVector.y = 0.;
+	BaseSurface.NewellSurfaceNormalVector.z = 1.;
+
+	SubSurface.Azimuth = 0.;
+	SubSurface.Tilt = 0.;
+	SubSurface.NewellSurfaceNormalVector.x = 0.;
+	SubSurface.NewellSurfaceNormalVector.y = 0.;
+	SubSurface.NewellSurfaceNormalVector.z = 1.;
+	checkSubSurfAzTiltNorm( BaseSurface, SubSurface, surfaceError );
+	EXPECT_FALSE( surfaceError );
+	EXPECT_TRUE( compare_err_stream( "" , true ) );
+
+	//Case 2 - Base surface and subsurface face the opposite way - should be error message and surfaceError=true
+	surfaceError = false;
+	SubSurface.Azimuth = 180.;
+	SubSurface.Tilt = 180.;
+	SubSurface.NewellSurfaceNormalVector.x = 1.;
+	SubSurface.NewellSurfaceNormalVector.y = 0.;
+	SubSurface.NewellSurfaceNormalVector.z = 0.;
+	checkSubSurfAzTiltNorm( BaseSurface, SubSurface, surfaceError );
+	EXPECT_TRUE( surfaceError );
+	//EXPECT_FALSE( compare_err_stream( "" , true ) );
```

Comment:
Oh duh - because without the EXPECT_EQ in compare_err_stream one won't see the two strings reported out when then differ.  So, my goal here of EXPECT_FALSE wrapped around compare_err_stream isn't going to work.  Would you be open to a function that simply returns the current err_stream?  With a reset option.